tests/app/views/test_admin.py

```diff
@@ -107,19 +106,10 @@ def test_can_update_alias(self):
     def test_creating_existing_index_updates_mapping(self):
```

Comment:
#ohISeeThen...

You need to rename this test. I mean, we are testing that it _doesn't_ update the mapping now, surely!